osf/models/metaschema.py

```diff
@@ -158,19 +159,16 @@ def absolute_api_v2_url(self):
 
 class RegistrationFormBlock(ObjectIDMixin, BaseModel):
     class Meta:
-        unique_together = ('schema', 'block_id')
         order_with_respect_to = 'schema'
 
     schema = models.ForeignKey('RegistrationSchema', related_name='form_blocks', on_delete=models.CASCADE)
-    page = models.CharField(max_length=255)
-    section = models.CharField(max_length=255, null=True)
     help_text = models.TextField()
-    block_id = models.CharField(max_length=255, db_index=True)
+    example_text = models.TextField(null=True)
+    answer_id = models.CharField(max_length=255, db_index=True, null=True)
+    chunk_id = models.CharField(max_length=24, db_index=True, null=True)
     block_type = models.CharField(max_length=31, db_index=True, choices=FORMBLOCK_TYPES)
-    block_text = models.TextField()
-    size = models.CharField(max_length=2, null=True, choices=FORMBLOCK_SIZES)
-    choices = ArrayField(models.TextField(), default=list)  # Longest on prod: >511 chars
-    required = models.BooleanField(default=True)
+    display_text = models.TextField()
+    required = models.BooleanField(default=False)
```

Comment:
changed `required` default to False - most blocks will always be false.  required is only relevant, where an answer_id exists (`-input blocks`) - this schema block which match the original question and will pull whether the question is required.